hoot-core/src/main/cpp/hoot/core/elements/MapProjector.cpp

```diff
@@ -430,8 +430,8 @@ bool MapProjector::_evaluateProjection(
   double width = env.MaxX - env.MinX;
   double height = env.MaxY - env.MinY;
   double stepSize = max(width, height) / 20.0;
-  int stepsX = (width) / stepSize;
-  int stepsY = (height) / stepSize;
+  double stepsX = (width) / stepSize;
+  double stepsY = (height) / stepSize;
   double stepSizeX = (width) / (double)stepsX;
   double stepSizeY = (height) / (double)stepsY;
```

Comment:
If `stepsX` is now a `double` there is no need to cast it to a `double` in these two lines.